providers/reported_at/provider_test.go

```diff
@@ -44,5 +44,5 @@ func TestReportedAtFiltering(t *testing.T) {
 	setup(t, RecommendedConfig)
 
 	testProvider.Info(context.Background(), false, "foo")
-	Expect(output.String()).To(MatchRegexp(`time=sometime level=info msg=foo reportedAt=".*go/src/testing/testing.go:\d+`))
+	Expect(output.String()).To(MatchRegexp(`time=sometime level=info msg=foo reportedAt=".*src/testing/testing.go:\d+`))
```

Comment:
is that related?

Reply:
I couldn't get the tests to pass without this

Reply:
This is what I was getting without this change

```
Expected
      <string>: time=sometime level=info msg=foo reportedAt="/usr/local/Cellar/go/1.11/libexec/src/testing/testing.go:830"

to match regular expression
      <string>: time=sometime level=info msg=foo reportedAt=".*go/src/testing/testing.go:\d+
```

Reply:
👍